### DISCUSSION TITLE
Improving player-exemple page design

### DISCUSSION THREAD

**Discussion by mickaelalvs** [remotion-dev/remotion#1856] [Ideas]
Doing two issues on the `player-example` package, I wonder if it was planned to add some style to this page ? One could quite simply add some styling to the page for the various buttons, toggles ... wyt @JonnyBurger ? 😊

**Comment by JonnyBurger** [comment]
For me it is just a technical testbed, where I can see if all the features are still working - but I wouldn't mind some styles if you want to contribute them!

Btw, thanks for your PR, hoping to review it today!

**Reply by mickaelalvs** [reply]
Yes indeed I think it's not an essential addition but if you want to create an issue to add a bit of style to have a more readable interface I'd love to take this issue ! You can ping me on it 😉

Btw very happy to have done this PR 🔥
